templates/auth_gssapi.conf.erb

```diff
@@ -3,7 +3,7 @@
   SSLRequireSSL
   AuthType GSSAPI
   AuthName "GSSAPI Single Sign On Login"
-  GssapiCredStore keytab:<%= scope.lookupvar('foreman::http_keytab') %>
+  GssapiCredStore keytab:<%= @http_keytab %>
```

Comment:
I was about to say that this should have broken https://github.com/theforeman/puppet-foreman/blob/91850a2b4522406a28dbc98e2c3f215130d08dab/spec/classes/foreman_config_ipa_spec.rb#L53

But we don't test this on Debian at all…
https://github.com/theforeman/puppet-foreman/blob/91850a2b4522406a28dbc98e2c3f215130d08dab/spec/classes/foreman_config_ipa_spec.rb#L5

Mind adding tests? :)

Reply:
I took the liberty of pushing some simple tests now ;)